rule/enum.go

```diff
@@ -32,5 +32,5 @@ func (r *enum) Validate(f, v, p string) (bool, error) {
 			return true, nil
 		}
 	}
-	return true, &ErrEnum{f, v, es}
+	return false, &ErrEnum{f, v, es}
```

Comment:
The `true` value on this line means that the `Validate` handler was executed fully but with validation error. There are another cases which the error is not related validation like this [piece of source code](https://github.com/guiferpa/gody/blob/main/rule/max_bound.go#L26-L35), taking a look at line 29 you noticed that there's a error about int conversion which it's not related validation logic directly, on this cases we return `false`.

Reply:
@The-iyed in this case we have to return `true` because `ErrEnum` is a validation error and not an unrelated error as mentioned above